common/common-ui/src/main/res/drawable/ic_tab_24_highlighted.xml

```diff
@@ -0,0 +1,16 @@
+<!--TODO this icon needs to be aligned with all the other new icons-->
```

Comment:
This will be fixed in https://app.asana.com/1/137249556945/project/715106103902962/task/1210170362960641?focus=true.